CFC_DataCollector/recommender/alternative_trips_pipeline.py

```diff
@@ -11,17 +15,30 @@ def __init__(self):
     def get_trips_for_alternatives(self, user_uuid):
         return ti.TripIterator(user_uuid, ["trips", "get_all"])
 
-    def runPipeline(self):
+    def runPipeline(self, immediate=False):
         for user_uuid in get_uuid_list():
+	    print "Finding Trips for User: ", uuid_to_name(str(user_uuid)), user_uuid
```

Comment:
We don't want to be associating trips with actual users or user email addresses.